modules/mobile/app/serializers/mobile/v0/check_in_serializer.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class CheckInSerializer
+      include JSONAPI::Serializer
+
+      set_type :check_in
+      attributes :id, :code, :message
+
+      def initialize(user_id, attributes)
+        @attributes = attributes
+
+        resource = CheckInStruct.new(id: user_id,
+                                     code:,
+                                     message:)
+        super(resource)
+      end
+
+      def code
+        case @attributes['message']
+        when 'success with appointmentIen: test-appt-ien, patientDfn: test-patient-ien, stationNo: test-station-no'
```

Comment:
The chip vcr's only show this message but the documentation that was already there shows other responses. I'm using a case statement in case we're going to need to account for other messages also. Will have to test in staging

Reply:
I would reach out in the check in slack channel to the engineers who wrote the service and see what they have to say.